packages/tocco-ui/src/StatedValue/StatedValue.js

```diff
@@ -41,8 +42,10 @@ const StatedValue = props => {
     touched
   } = props
 
+  const adjustedLabel = js.adjustedHTMLString(label)
```

Comment:
why do you think we cannot interpret the labels als HTML? aren't they all coming from our language properties files?

Reply:
you are right. since this is a trusted source we should be allowed to that. i changed it